src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -1523,14 +1523,16 @@ class ChangesClauseSegment(BaseSegment):
             "INFORMATION",
             Ref("ParameterAssignerSegment"),
             OneOf("DEFAULT", "APPEND_ONLY"),
+            parse_mode=ParseMode.GREEDY,
```

Comment:
Why are these needed now out of interest?

Reply:
Not needed at all, then just mean slightly more specific parsing errors if snowflake change the syntax again. I'm guessing some of those keywords might change and if they do - only the bracketed section that has changed will fail and we'll get a faster report on the issue.